.github/workflows/push-checks.yml

```
@@ -92,6 +92,11 @@ jobs:
   test-e2e:
     name: Test - end-to-end
     needs: [test]
+    strategy:
+      matrix:
+        type:
```

This should probably be renamed to something that makes more sense, e.g.:

```suggestion
        svgo-version:
```